### DISCUSSION TITLE
Welcome to YAMACS!

### DISCUSSION THREAD

**Discussion by YAMACS-SML** [YAMACS-SML/YAMACS#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together.

  Please share with us your questions, issues, and suggestions to improve our plugins and let's grow this community together! 💪

**Comment by chaoyangma24** [comment]
When I run protein-lig complex, there appears "the plugin crashed with RuntimeError: YASARA raised error 103: Unknown command or syntax error. if there is no typing mistake, and no indentation mistake when providing parameters for an experiment, this command requires either a newer version or higer stage of YASAR." But my YASARA version (view) is the newest (version 22.9.24). I want to know why? Thank for you!

**Reply by YAMACS-SML** [reply]
Hello chaoyangma24, 
Thanks for using YAMACS and sharing your valuable feedback. To resolve this unwanted error we need more information about it. It would be appreciated if you can share any image of that error as well as the  image of the command prompt (both Yasara and Linux command prompt).Then we can provide a proper resolution about this error.
Thank you
